### DISCUSSION TITLE
Can minor releases of OpenCRX be expected in the near future?

### DISCUSSION THREAD

**Discussion by javac9** [opencrx/opencrx#73] [Q&A]
Kudos for moving onto the GitHub! I guess it would be really really great to see more frequent updates (minor at least, eg. 4.3.1, etc.) of OpenCrx in the future, as opposed to waiting for major releases for longer periods of time. I guess that would make your lives easier as well - fixing minor issues as soon as they are reported by us users.

What is your position on this?

**Comment by wfro64** [comment]
Yes, in the future it is planned to publish minor and bug-fix releases.

See https://github.com/opencrx/opencrx/releases
